src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/dashboard/scenarios/claimant/CCJRequestedScenarioTest.java

```diff
@@ -30,7 +30,6 @@ void should_create_ccj_requested_scenario() throws Exception {
             .toBuilder()
             .legacyCaseReference("reference")
             .ccdCaseReference(Long.valueOf(caseId))
-            .respondent1ResponseDeadline(responseDeadline.atStartOfDay())
```

Comment:
why was this removed?

Reply:
Added back

Reply:
This comes from Max PR, I think we should check it with him @m-meulendijks-v1

Reply:
According to Max, it wasn't relevant to the test, so it is removed.
@neeta-hmcts @portizdev